src/mono/netcore/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -661,7 +662,8 @@
 		<!-- domain.c: mono_defaults.thread_class -->
 		<!-- FIXME: -->
 		<type fullname="System.Threading.Thread" preserve="fields">
-			<method name="get_CurrentContext" />
+      <!-- TODO: Not sure what this was in Mono - doesn't exist in NetCore -->
+			<!--<method name="get_CurrentContext" />-->
```

Comment:
Looks like https://docs.microsoft.com/en-us/dotnet/api/system.threading.thread.currentcontext?view=netframework-4.8 which didn't make it over to netcore like you said.